src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/lstm_cell_basic.cpp

```diff
@@ -23,14 +23,28 @@ float clip = 0.f;
 std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
                                                          InferenceEngine::Precision::FP16};
 
+// LSTMCell is decomposed to Convolution.
+// Before GNA_TARGET_3_5 weights used I16 precision only even if GNA_PRECISION was set to I8.
+// Since GNA_TARGET_3_5 weights with precision I8 are allowed for convolutions,
+// but proper results are expected for models after POT.
+//
+// This is related to the issue 70675.
 std::vector<std::map<std::string, std::string>> configs = {{{"GNA_DEVICE_MODE", "GNA_SW_FP32"}},
                                                            {{"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
+                                                            {"GNA_EXEC_TARGET", "GNA_TARGET_3_0"},
+                                                            {"GNA_PRECISION", "I8"},
+                                                            {"GNA_SCALE_FACTOR_0", "1024"},
```

Comment:
if possible, please use separate configurations to set common test parameters and target specific